inputfiles/knownTypes.json

```diff
@@ -30,6 +31,13 @@
         "NamedCurve",
         "Pbkdf2Params",
         "PropertyIndexedKeyframes",
+        "RsaHashedImportParams",
+        "RsaHashedKeyAlgorithm",
+        "RsaHashedKeyGenParams",
+        "RsaKeyAlgorithm",
+        "RsaKeyGenParams",
+        "RsaOaepParams",
+        "RsaPssParams",
```

Comment:
I think this one is initially sorted by `.sort()` and thus case-sensitive, but probably okay.